libs/data/tests/test_distributions.py

```diff
@@ -57,3 +61,19 @@ def test_log_normal():
     samples = sampler(10000)
     mean = samples.mean().item()
     assert (abs(mean - 10) / 10) < (2 * 2 / 10000**0.5)
+
+
+def test_volume_distributed_snr():
+    """Test VolumeDistributedSNR against expected ideal distribution"""
+    ref_snr = 8
+    sampler = distributions.VolumeDistributedSNR(ref_snr)
+    samples = sampler(10000).numpy()
+    # check 1/rho^4 behavior
+    counts, ebins = np.histogram(samples, bins=100)
+    bins = ebins[1:] + ebins[:-1]
+    bins *= 0.5
+
+    foo = lambda x, a, b: a * x ** (-b)  # noqa E731

```

Comment:
I find this neater. Shall we consider this an exception, since its a test script anyway.

Reply:
Given that it's just a test script and that it's obvious that this function won't raise an error, I would say keeping it is fine for now. But in general in the future I think the PEP [programming guidelines](https://peps.python.org/pep-0008/#programming-recommendations) make a good case why this syntax is worth avoiding. Namely

> The (`def`) form means that the name of the resulting function object is specifically (‘foo’) instead of the generic ‘<lambda>’. This is more useful for tracebacks and string representations in general. The use of the assignment statement eliminates the sole benefit a lambda expression can offer over an explicit def statement (i.e. that it can be embedded inside a larger expression)

Reply:
ok.

Reply:
Huh ok I did not realize that `black` didn't allow this one-line `def` syntax. Kind of weird frankly but good to know, still probably more clear but I get the appeal of the lambda formulation now

Reply:
nevermind. I used the PEP syntax.